airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -244,7 +244,7 @@ def get_link(
         :param dttm: datetime
         :return: url link
         """
-        if ti_key:
+        if ti_key is not None:
```

Comment:
This makes no difference does it? (since we don't define a custom book on TIKey)

Reply:
Nope. But it's easier to standardize. Both ways were used in the code and I arbitrary choose `not None` as more explicit.

Reply:
This way we have only one "standard" way to check.